recipes/MOABB/train.py

```diff
@@ -131,6 +139,28 @@ def on_stage_end(self, stage, stage_loss, epoch=None):
                     test_stats=last_eval_stats,
                 )
 
+    def check_if_best(
```

Comment:
Here you save the best model for at least one of the specified keys (which are the default ones following your call at line 121 of train.py). Once you have completed the training and you perform the evaluation, the loaded model is not the best model for the 'test_keys' specified in the yaml file. Should we change the call of this method (specifying 'keys=self.hparams.test_keys') or remove the 'test_keys' parameter in the yaml file?

Reply:
seems like the first option was the initial idea when the test_keys was added in the yaml file